src/helpers/helpers.sh

```diff
@@ -43,6 +43,7 @@ if [ "$(is_git_repository)" == true ]; then
   export INITIAL_BRANCH_NAME=$(get_current_branch_name)
   export INITIAL_DIRECTORY=$(pwd)
   export INITIAL_OPEN_CHANGES=$(has_open_changes)
+  export INITIAL_PREVIOUS_BRANCH_NAME=$(get_previous_branch_name)
```

Comment:
Yeah, I also think the parity with `INITIAL_` is important if only to keep the naming consistent with the purpose of recording what the inital state was.